crates/pop-cli/src/commands/up/parachain.rs

```diff
@@ -115,7 +150,7 @@ impl UpChain {
 
 	// Prepares and returns the encoded call data for registering a parachain.
 	fn prepare_register_parachain_call_data(&self) -> Result<Vec<u8>> {
-		let UpChain { id, genesis_code, genesis_state, chain } = self;
+		let UpChain { id, genesis_code, genesis_state, chain, proxy } = self;
 		let ex = find_dispatchable_by_name(
```

Comment:
`let register` might be more informative

Reply:
In the main feat branch name was changed to `dispatachable`